src/System.IO.FileSystem/src/System.IO.FileSystem.builds

```diff
@@ -10,22 +10,13 @@
     </Project>
     <Project Include="System.IO.FileSystem.csproj">
```

Comment:
I had to keep those for the package projects since they still build the csproj for netstandard1.3/netcore50 (see above comment). Alternatively, I could remove those as well but @weshaggard mentioned that those may need to remain. I'm not sure what the purpose of creating packages for ns1.3/netcore50 is.